python/tests/test_account.py

```diff
@@ -1868,6 +1869,44 @@ def update_name():
             # No renames should happen after explicit rename
             assert updated_name == "Renamed"
 
+    @pytest.mark.parametrize("mvbox_move", [False, True])
+    def test_add_all_receipients_as_contacts(self, acfactory, lp, mvbox_move):
+        """Delta Chat reads the receipients from old emails sent by the user user and add them as contacts.
+        This way, we can already offer them some email addresses they can write to.
+
+        Also test that existing emails are fetched during onboarding.
+
+        Lastly, tests that bcc_self messages moved to the mvbox are marked as read."""
+        ac1 = acfactory.get_online_configuring_account(mvbox=mvbox_move, move=mvbox_move)
+        ac2 = acfactory.get_online_configuring_account()
+
+        acfactory.wait_configure_and_start_io()
+
+        chat = acfactory.get_accepted_chat(ac1, ac2)
+
+        lp.sec("send out message with bcc to ourselves")
+        if mvbox_move:
+            ac1.direct_imap.select_config_folder("mvbox")
+        ac1.direct_imap.idle_start()
+        ac1.set_config("bcc_self", "1")
+        chat.send_text("message text")
+        assert ac1.get_config("bcc_self") == "1"
```

Comment:
I added it because I had problems with this value changing (because of https://github.com/deltachat/deltachat-core-rust/blob/1ae5020efe1756fe9b94f6f8476f14244c751cab/python/src/deltachat/testplugin.py#L358 from https://github.com/deltachat/deltachat-core-rust/commit/07236efc45a5c5c10ef64c0978e264adff8428a6; I guess that @hpk42 did't see any other solution) but I'll remove it, it's not needed anymore.